SampleAuth.php

```diff
@@ -55,14 +62,29 @@ function auth_user_flags( $p_event_name, $p_args ) {
 			return null;
 		}
 
-		$t_access_level = user_get_access_level( $t_user_id, ALL_PROJECTS );
+		if( $t_user_id ) {
```

Comment:
Should the above `user_is_anonymous()` be inside this if statement?

Reply:
I don't think so, and I really don't see any point getting the user_access_level if it's not really needed.